example/src/PagerHookExample.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo } from 'react';
+import { StyleSheet, View, SafeAreaView, Text } from 'react-native';
+
+import { LikeCount } from './component/LikeCount';
+import { NavigationPanel } from './component/NavigationPanel';
+import { useViewPager } from 'react-native-pager-view';
+
+export function PagerHookExample() {
+  const { AnimatedPagerView, ref, ...rest } = useViewPager(10);
```

Comment:
I think in order to make this scalable and not introduce breaking changes in the future, a better design will be to use an object:

```ts
const { AnimatedPagerView, ref, ...rest } = useViewPager({pageCount: 10});
```

This way we can easily add more parameters later